backends/circuit-breaker.md

```diff
@@ -48,33 +80,5 @@ And this is the way the states change:
 | ![Krakend logo](/images/documentation/circuit-breaker-states.png) |
 
 - `CLOSED`: In the initial state, the system is healthy and sending connections to the backend.
-- `OPEN`: When a consecutive number of supported errors from the backend (`maxErrors`) is reached, the system changes to `OPEN` and no further connections are sent to the backend. The system will stay in `OPEN` state for N seconds ( the `timeout`).
+- `OPEN`: When a consecutive number of supported errors from the backend (`maxErrors`) is reached, the system changes to `OPEN`, and no further connections are sent to the backend. The system will stay in `OPEN` state for N seconds ( the `timeout`).
```

Comment:
When a consecutive number of supported errors from the backend (`maxErrors`) is exceeded